pkg/offline/offline.go

```diff
@@ -5,16 +5,40 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
+	"path"
 
 	"github.com/wakatime/wakatime-cli/pkg/heartbeat"
 
+	_ "github.com/mattn/go-sqlite3" // not used directly
+	"github.com/mitchellh/go-homedir"
 	jww "github.com/spf13/jwalterweatherman"
 )
 
 const (
 	tableName = "heartbeat_2"
 )
 
+// QueueFilepath returns the path to the offline queue db file.
+func QueueFilepath() (string, error) {
+	dir := os.Getenv("WAKATIME_HOME")
+
+	var err error
+	if dir == "" {
+		dir, err = os.UserHomeDir()
+		if err != nil {
+			return "", fmt.Errorf("failed to retrieve user's home dir: %s", err)
```

Comment:
just to follow the other messages like that it should be `failed getting user's home directory: %s`

Reply:
Actually the other messages in the package are all in this format (`failed to ...`). Can you point me to where we use the other format?